### PR TITLE
Pull request for Client directory

### PR DESCRIPTION
**Pull request by noobed-max**
- Completed prerequisites of Client directory

### CODE REVIEW

File: client/s1_client/crud.py
Comment (Tejas-ChandraShekarRaju) [inline client/s1_client/crud.py:1]: Why is this removed?

File: client/s1_client_test/client.py
Comment (Tejas-ChandraShekarRaju) [inline client/s1_client_test/client.py:1]: Why is all this added in s1_client_test? Keep everything in s1_client

File: client/s1_client_test/crud.py
Comment (Tejas-ChandraShekarRaju) [inline client/s1_client_test/crud.py:1]: What is this? Why is there a database file in client folder?

File: client/setup.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
 from setuptools import setup, find_packages
 
 setup(
-    name='s1_client',
-    version='0.1.0',
+    name='s1_client_test',
```
Comment (Tejas-ChandraShekarRaju) [inline client/setup.py:4]: Change to ciaobjectstore

File: client/s1_client_test/crud.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+
```
Comment (Tejas-ChandraShekarRaju) [inline client/s1_client_test/crud.py:1]: Why is the changes to server coming in this commit?

File: .gitignore
Comment (Tejas-ChandraShekarRaju) [inline .gitignore:1]: @noobed-max  I just need one commit. And a properly structured client repo with save,retrieve,update and delete functionalities in it.

**Review by Tejas-ChandraShekarRaju — CHANGES_REQUESTED** [review]
Please follow my file structure and add in files that are currently not there.

File: client/setup.py (hunk 1 above)
Comment (noobed-max) [reply client/setup.py:4]: change name to ciastore right?
that measn i have to change the folder name s1_client to ciastore

Comment (Tejas-ChandraShekarRaju) [reply client/setup.py:4]: yeah. If ciaos is available then go with ciaos

File: client/s1_client/crud.py
Comment (noobed-max) [reply client/s1_client/crud.py:1]: we dont need crud in client , as its on the backend server
